### PR TITLE
feat: add support for Kubernetes v1.16.15

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:
See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#changelog-since-v11614

**Issue Fixed**:

**Requirements**:

- [x] Kubernetes artifacts built and pushed by Azure Pipelines
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

v1.16.15 is the last patch release for the 1.16 series.

### CODE REVIEW

**Comment by mboersma** [comment]
/hold

This has a temporary commit for testing that needs to be discarded before merging.

**Comment by jackfrancis** [comment]
@jadarsie heads up about an Azure Stack build

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Pull request merged**
